README.md

```diff
@@ -100,18 +100,18 @@ docker cp "$container_id:/usr/local/lib/python3.6/dist-packages/deepcell/utils/c
 docker kill $container_id
 
 # you can now start the docker image with the code mounted for easy editing
-NV_GPU='0' nvidia-docker run -it \
+docker run --gpus '"device=0"' -it \
     -p 8888:8888 \
     -v $PWD/deepcell:/usr/local/lib/python3.6/dist-packages/deepcell/ \
     -v $PWD/scripts:/notebooks \
     -v /$PWD:/data \
-    $USER/deepcell-tf:0.4.0-gpu
+    $USER/deepcell-tf:latest
```

Comment:
Same here, IDK what latest is, maybe it has GPU support, maybe it does not. let's stick to versioned releases.